.eslintrc.js

```diff
@@ -0,0 +1,145 @@
+const ERROR = 2;
+
+const OFF = 0;
+
+const COUNT = 1;
+
+const REACT_RULES = {
+  'react/no-children-prop': ERROR,
+  'react/no-deprecated': ERROR,
+  'react/no-multi-comp': ERROR,
+  'react/no-string-refs': ERROR,
+  'react/no-unescaped-entities': ERROR,
+  'react/no-unstable-nested-components': ERROR,
+  'react/self-closing-comp': ERROR,
+  'react/jsx-boolean-value': [ERROR, 'always'],
+  'react/jsx-fragments': ERROR,
+  'react/jsx-handler-names': [
+    ERROR,
+    {
+      checkInlineFunction: true,
+      checkLocalVariables: true,
+    },
+  ],
+  'react/jsx-key': ERROR,
+  'react/jsx-no-leaked-render': ERROR,
+  'react/jsx-no-useless-fragment': ERROR,
+  'react/jsx-sort-props': ERROR,
+};
+
+const ESLINT_RULES = {
+  'no-console': ERROR,
+  'object-shorthand': ERROR,
+  'no-magic-numbers': [ERROR, { ignoreArrayIndexes: true, ignore: [1, 0, -1] }],
+};
+
+const TYPESCRIPT_RULES = {
+  '@typescript-eslint/no-unused-vars': [
+    ERROR,
+    {
+      argsIgnorePattern: '^_',
+      varsIgnorePattern: '^_',
+    },
+  ],
+  '@typescript-eslint/no-var-requires': OFF,
+  '@typescript-eslint/no-explicit-any': [ERROR],
+};
+
+const IMPORT_RULES = {
+  'import/newline-after-import': [ERROR, { count: COUNT }],
+  'import/export': OFF,
+  'import/order': [
+    ERROR,
+    {
+      groups: [
+        'builtin',
+        'external',
+        'parent',
+        'sibling',
+        'internal',
+        'index',
+        'object',
+      ],
+      pathGroups: [
+        {
+          pattern: '{react,react-native,recoil,recoil-nexus,}',
+          group: 'external',
+          position: 'before',
+        },
+        {
+          pattern: '{components/**,modules,components/**}',
+          group: 'parent',
+          position: 'before',
+        },
+        {
+          pattern: '{screens/**,modules,screens/**}',
+          group: 'parent',
+          position: 'before',
+        },
+        {
+          pattern: '{assets/**,constants/**}',
+          group: 'parent',
+          position: 'before',
+        },
+        {
+          pattern: '{helpers/**,services/**,navigators/**,store/**,}',
```

Comment:
here, too, for beauty, you can remove the comma after store/**

Reply:
As ESLintrc has a JS extension, we follow our ESLint rules in this file, we have prettier regulations, and one of these rules is about the last comma in the object.

[rule](https://github.com/prettier/eslint-plugin-prettier#options):